documents/order.md

```diff
@@ -275,6 +275,75 @@ Razorpay::Order.edit(orderId,para_attr)
 -------------------------------------------------------------------------------------------------------
 
 
+### View RTO/Risk Reasons
+
+```go
+orderId = "order_XXXXXXXXXXXXX1"
+
+Razorpay::Order.fetch(orderId).view_rto
+```
+**Parameters**
+
+| Name     | Type   | Description                         |
+|----------|--------|-------------------------------------|
```

Comment:
please format the table

Reply:
formatted

https://github.com/razorpay/razorpay-ruby/blob/1fcd3fa00ae1048e83a282d329e3f3d0bdf6b1cb/documents/order.md?plain=1#L287